simple-facebook-instant-articles.php

```diff
@@ -151,6 +165,38 @@ public function feed_template() {
 	}
 
 	/**
+	 * Set WP query variables for FB IA feed, so we can customise
+	 * what posts are considered for the feed.
+	 *
+	 * @param $query WP_Query object.
+	 */
+	public function customise_feed_query( $query ) {
+
+		$feed_slug = apply_filters( 'simple_fb_feed_slug', $this->token );
+
+		// FB IA feed - customise query.
+		if ( $query->is_main_query() && $query->is_feed( $feed_slug ) ) {
+
+			// Number of posts for feed.
+			$num_posts = intval( apply_filters( 'simple_fb_posts_per_rss', get_option( 'posts_per_rss', 10 ) ) );
+			$query->set( 'posts_per_rss', $num_posts );
+
+			// Exclude sponsored articles.
+			$exclude_sponsored_posts = array(
+				array(
+					'key'     => '_format_sponsored_link',
+					'value'   => '',
+					'compare' => '=',
```

Comment:
Yes I think its mostly fine to check for empty string... but I'd like to not rely on that if possible. If we want to allow post with nothing set for sponsored_link, then we should check for NOT EXISTS as well